e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/PipelineLibraryContentAO.java

```diff
@@ -102,6 +104,10 @@ public PipelineLibraryContentAO assertVersion(String version) {
         return ensure(FIRST_VERSION, text(version));
     }
 
+    public PipelineLibraryContentAO assertPipelineName(String name){
+        return ensure(xpath("//div[@class = 'browser__item-header']//span"),hasText(name));
```

Comment:
Add a space between ',hasText(name)', please. And let's use the 'text()' method instead

Reply:
Done